src/track/beatutils.cpp

```diff
@@ -12,15 +12,17 @@
 #include <QMap>
 
 #include "track/beatutils.h"
+#include "track/beatstats.h"
 #include "util/math.h"
 
 // we are generous and assume the global_BPM to be at most 0.05 BPM far away
 // from the correct one
-#define BPM_ERROR 0.05
+constexpr double kBpmError = 0.05;
+constexpr double kMaxPhaseError = 0.025; //25ms
```

Comment:
Can you include the unit in the name? The we don't need the comment and also have that information where it's used.